### PR TITLE
chore: remove BigQuery cloud-client samples

### PR DESCRIPTION
**Pull request by plamut**
Do not merge until https://github.com/googleapis/python-bigquery/pull/148, as well as an internal PR that will redirect the BigQuery docs to point to the new samples location (TODO: somebody please open that PR).

### CODE REVIEW

**Comment by plamut** [comment]
Samples have been moved to the BigQuery repo: https://github.com/googleapis/python-bigquery/pull/174

We need to point the docs at cloud.google.com to the new location before merging this.

**Comment by shollyman** [comment]
I'll work on doing the docsite internal change today to re-point all the embeds.

**Comment by shollyman** [comment]
tracking internally as 162000215

File: bigquery/cloud-client/README.rst [hunk 1]
```diff
@@ -1,149 +1,3 @@
-.. This file is automatically generated. Do not edit this file directly.
+These samples have been moved.
```
Comment (tmatsuo) [inline bigquery/cloud-client/README.rst:1]: Can we add where it got moved?

Comment (busunkim96) [reply bigquery/cloud-client/README.rst:1]: It looks like this is accounted for (see the last line of this file)

Comment (plamut) [reply bigquery/cloud-client/README.rst:1]: Indeed, the new URL is at the end.

Perhaps it wasn't that apparent in the unified diff view? (a side-by-side diff renders it much more clearly IMHO)

**Comment by plamut** [comment]
What's the status of this, can somebody please re-run the tests (they appeared stalled)? Thanks!

**Pull request merged**